### ISSUE TITLE
get cost stats for each Azure OpenAI calls

### ISSUE THREAD

**Issue by GuillaumeTurcotte** [dto-btn/chatbot#2]
We need to present some stats to the higher manager on what is the average cost of a chat question.

**Comment by GuillaumeTurcotte** [comment]
According to this [pricing page](https://azure.microsoft.com/en-ca/pricing/details/cognitive-services/openai-service/) and using our current index with a query like this one : "What records management system does SSC use?" the following token usage was the following:

```bash
Loading all indices.
INFO:llama_index.token_counter.token_counter:> [retrieve] Total LLM token usage: 0 tokens
> [retrieve] Total LLM token usage: 0 tokens
INFO:llama_index.token_counter.token_counter:> [retrieve] Total embedding token usage: 9 tokens
> [retrieve] Total embedding token usage: 9 tokens
INFO:llama_index.token_counter.token_counter:> [get_response] Total LLM token usage: 1642 tokens
> [get_response] Total LLM token usage: 1642 tokens
INFO:llama_index.token_counter.token_counter:> [get_response] Total embedding token usage: 0 tokens
> [get_response] Total embedding token usage: 0 tokens
INFO:llama_index.token_counter.token_counter:> [get_response] Total LLM token usage: 1642 tokens
> [get_response] Total LLM token usage: 1642 tokens
INFO:llama_index.token_counter.token_counter:> [get_response] Total embedding token usage: 0 tokens
> [get_response] Total embedding token usage: 0 tokens
```

Which is a total of  3284 tokens for the LLM (`chatgpt-3.5-turbo`) and 9 tokens for the embedding.

This amount to `$0.002729 CAD x (3600/1000) tokens` in average for the LLM:  $0.01 CAD (rounded) for LLM.
And for the ADA (embedding searches) it's `$0.000546 x 9/1000 tokens` in average for the Embedding: $0.000004914 CAD

In summary average cost per questions would be  `$0.01 + $0.000005` CAD.
